it.hjson

```diff
@@ -161,66 +161,66 @@
   "generic.featureAlwaysAllowedInPrivate": "%s è sempre consentito in privato :)"
 
   # ℹ️ Example: <b>Wall Building has now been enabled in this group.</b>
-  "generic.featureEnabled": "<b>%s è stato ora abilitato in questo gruppo</b>"
+  "generic.featureEnabled": "<b>%s è ora abilitato in questo gruppo</b>"
   "generic.featureAlreadyEnabled": "<b>%s è già abilitato in questo gruppo</b>"
   "generic.featureDisabled": "<b>%s è ora disabilitato in questo gruppo</b>"
   "generic.featureAlreadyDisabled": "<b>%s è già disabilitato in questo gruppo</b>"
 
   # ℹ️ Example: By default, fishing is disabled in group chats.</b>\n\nAn admin can /EnableFish, or you can fish in private, by tapping the button below:
-  "generic.featureDisabledByDefault": "<b>Per impostazione predefinita, %s è disabilitato nelle chat di gruppo.</b>\n\nUn amministratore può %s oppure puoi %s in privato, toccando il pulsante qui sotto:"
+  "generic.featureDisabledByDefault": "<b>Per impostazione predefinita, %s è disabilitato nelle chat di gruppo.</b>\n\nUn amministratore può dare %s oppure puoi %s in privato, usando il pulsante qui sotto:"
 
   # ℹ️ Example: <b>You can use </b>/DisableBrick<b> to change that.</b>
-  "generic.toggleFeatureHint": "<b>Puoi usare </b>%s<b> per cambiarlo</b>"
+  "generic.toggleFeatureHint": "<b>Puoi usare </b>%s<b> per modificarlo</b>"
 
   # ℹ️ Example: You can't Warn admins
   "generic.cantUseFeatureOnAdmins": "<b>Non puoi </b><code>%s</code><b> amministratori... Purtroppo.</b>"
   "generic.cantUseFeatureOnAdminsWithReason": "<b>Non puoi </b><code>%s</code><b> amministratori perché %s</b>"
   # ℹ️ Example: because they can't be warned in the first place.
-  "generic.theyCantBeInTheFirstPlace": "perché non possono essere %s innanzitutto."
+  "generic.theyCantBeInTheFirstPlace": "perché non possono essere %s comunque."
 
 
   /* ---------------------------------------------------------------------
                       Comfy - Used for /hug, /cuddle, etc.
      --------------------------------------------------------------------- */
   "comfy.hug": {
-    "self": "%s<b> raggiunge un braccio intorno e abbraccia il proprio corpo. </b>%s<b> si sente leggermente meno solo ora.</b>"
-    "main": "%s ti ho appena dato un <b>abbraccio! :)</b>"
+    "self": "%s<b> porta un braccio intorno a sé stess* e si abbraccia da sol*. </b>%s<b> si sente leggermente meno sol*.</b>"
+    "main": "%s ti ha appena <b>abbracciat*! :)</b>"

```

Comment:
Are the added * on these lines here for a reason?

There are more on some other translations below

```suggestion
    "main": "%s ti ha appena <b>abbracciat! :)</b>"
```

Reply:
Unfortunately Italian is not gender neutral, so it can be abbracciato (masc.) or abbracciata (fem.), hence why the "*" cheat, or "@" as Spanish speakers do.

Reply:
Ah, clever. Let's keep these then 👍